src/components/FooterBar/FooterBar.tsx

```diff
@@ -43,14 +43,21 @@ const getData = (configFile?: ConfigFile) => {
     {
       category: "Utilities",
       items: [
-        { label: "Verify Documents", to: `${networkPath}/verify`, render: renderExternalLinkWithoutIcon },
+        { label: "Verify Documents", to: `${networkPath}/verify`, render: renderExternalLink },
         { label: "Create Documents", to: "/", render: renderNavLink },
       ],
     },
     {
       category: "Resources",
       items: [
-        { label: "Webinars", to: `${networkPath}/learn`, render: renderExternalLinkWithoutIcon },
+        { label: "Learn", to: `${networkPath}/learn`, render: renderExternalLinkWithoutIcon },
+        { label: "FAQ", to: `${networkPath}/faq`, render: renderExternalLinkWithoutIcon },
+        { label: "ETA", to: `${networkPath}/eta`, render: renderExternalLinkWithoutIcon },
```

Comment:
question (non-blocking): there are only a few internal links. should we change external links from `renderExternalLinkWithoutIcon` to  `renderExternalLink`? (so as to add the icon)

perhaps to discuss if it's still make sense to make creator "look" like part of tradetrust website via similar header, footer (we know that they are separate sites) <- there wasn't any conclusion iirc

Reply:
ok can i was thinking of changing to externalLink too.

but whats the issue with creator to "look" like part of tradetrust website? isn't it suppose to be like that?

Reply:
i think there was once mentioned to drop the header navbar and footer links, reverting to the previous stance of a standalone application, due to the confusion of leading off to tradetrust website (ux concern)

we can leave it as it is first, yeah ok with `externalLink`